homeassistant/components/kostal_plenticore/select.py

```diff
@@ -12,22 +12,29 @@
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
+from ...helpers.entity_platform import AddEntitiesCallback
```

Comment:
It's better to keep all non intra package imports absolute to stay compatible if the integration is run as a custom integration.

Reply:
fixed